CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Added
 
+- Added support for non-list types in *Object Data Path Output* node.
```

Comment:
Initially, I wanted to fix the [`Cannot set attribute`](https://github.com/rassvet2/animation_nodes/blob/patch-1/animation_nodes/nodes/object/object_data_path_output.py#L26) error, so maybe the Fixed section is better, but it's also a feature addition……